pages/blogs.tsx

```diff
@@ -1,14 +1,89 @@
+import commerce from '@lib/api/commerce';
+import { GetStaticPropsContext } from 'next';
 import { Layout } from 'src/components/common';
-import { BlogsList, FeaturedCardBlog, BlogHeading, BlogBreadCrumb } from 'src/components/modules/blogs';
+import { BlogCardProps } from 'src/components/common/CardBlog/CardBlog';
+import { BlogBreadCrumb, BlogHeading, BlogsList, FeaturedCardBlog } from 'src/components/modules/blogs';
+import { DEFAULT_BLOG_PAGE_SIZE } from "src/utils/constanst.utils";
+import { getAllPromies } from 'src/utils/funtion.utils';
+import { PromiseWithKey } from 'src/utils/types.utils';
 
-export default function BlogsPage() {
+interface Props {
+    blogsResult: { blogs?: BlogCardProps[],featuredBlog?: BlogCardProps[] },
```

Comment:
seperate blogs and featureBlogs
```suggestion
blogs?: BlogCardProps[]
featuredBlog?: BlogCardProps[]
```